homeassistant/components/seventeentrack/sensor.py

```diff
@@ -58,16 +60,17 @@
         vol.Required(CONF_PASSWORD): cv.string,
         vol.Optional(CONF_SHOW_ARCHIVED, default=False): cv.boolean,
         vol.Optional(CONF_SHOW_DELIVERED, default=False): cv.boolean,
+        vol.Optional(CONF_TIMEZONE, default="UTC"): cv.string,
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

https://developers.home-assistant.io/docs/config_entries_config_flow_handler

Reply:
Why can't we just use the timezone of Home Assistant?

Reply:
> Why can't we just use the timezone of Home Assistant, `hass.config.time_zone`?

Yes, good idea. Will be close to zero instances where a user would want to choose a timezone different than that.